src/models/discriminant_analysis.jl

```diff
@@ -537,34 +572,37 @@ Train the machine using `fit!(mach, rows=...)`.
 
 The fields of `fitted_params(mach)` are:
 
-- `projected_class_means`: The matrix comprised of class-specific means as columns, of size
-  `(indim, nclasses)`, where `indim` is the number of input features (columns) and
-  `nclasses` the number of target classes.
+- `classes`: The classes seen during model fitting.
 
 - `projection_matrix`: The learned projection matrix, of size `(indim, outdim)`, where
- `indim` and `outdim` are the input and output dimensions respectively.
+  `indim` and `outdim` are the input and output dimensions respectively (See Report 
+  section below).
 
 # Report
 
 The fields of `report(mach)` are:
 
-- `classes`: The classes seen during model fitting.
+- `indim`: The dimension of the input space i.e the number of features of training matrix.
```

Comment:
Maybe "training matrix" is a little confusion as use must supply a table, right?

```suggestion
- `indim`: The dimension of the input space i.e the number of training features.
```